doc/build_and_run.md

```diff
@@ -47,7 +29,15 @@ Refer to [this guide](https://github.com/NOAA-EMC/rrfs-workflow/wiki/deploy-a-re
 ./setup_rocoto.py exp.conus12km
 ```   
     
-This Python script creates an experiment directory (i.e. `EXPDIR`), writes out a runtime version of `exp.setup` under EXPDIR, and  then copies runtime config files from `HOMErrfs/parm` to `EXPDIR`.
+This Python script creates an experiment directory (i.e. `EXPDIR`), writes out a runtime version of `exp.setup` under EXPDIR, and  then copies runtime config files to `EXPDIR`.  
+
+If you get errors when running `setup_rocoto.py`, it is mostly because currently loaded Python is lower than expected.  
+You may run 
+```
+source ../workflow/ush/load_bokeh.sh
```

Comment:
what does "bokeh" mean?

Reply:
Good question. `bokeh` is a name for a Python environment based on Miniforge3.
The word itself means "In photography, bokeh refers to the aesthetic quality of the blur in an image, particularly in out-of-focus areas like the background or foreground." 
The name comes from a famous Python package called "Bokeh"